### PR TITLE
New admin labels for IN

### PR DESCRIPTION
**Pull request by stepps00**
Fixes the India portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1640

This PR probably needs a second pair of eyes to verify the Hindi translations, if possible. The translations look correct from sources I can find online, but the order of placetype + name in Hindi is a bit tricky with States and Union Territories.

India has English and Hindi as "official" languages, so this PR updates English and Hindi label properties on region and county records. In some cases, WOF does not have a Hindi translation for the county, so only English was added.

There are over a dozen "spoken" languages in India that I can work through to add labels, but we'll have to prioritize which languages to add (I'll have to curate new values).

No PIP work, can merge once approved.

### CODE REVIEW

File: data/110/869/570/1/1108695701.geojson [hunk 1]
```diff
@@ -10,6 +10,18 @@
...
     "geom:longitude":76.509741,
     "iso:country":"IN",
+    "label:eng_x_preferred_longname":[
+        "Kargil District"
+    ],
+    "label:eng_x_preferred_placetype":[
+        "district"
+    ],
+    "label:hin_x_preferred_longname":[
+        "\u0915\u093e\u0930\u0917\u093f\u0932 \u091c\u093c\u093f\u0932\u093e \u091c\u093c\u093f\u0932\u093e"
```
Comment (nvkelso) [inline data/110/869/570/1/1108695701.geojson]: Looks like the district text must be in the `name`, so check to make sure it's not repeating.

File: data/110/880/597/9/1108805979.geojson [hunk 2]
```diff
@@ -17,6 +17,12 @@
     "label:eng_x_preferred_placetype":[
         "state"
     ],
+    "label:hin_x_preferred_longname":[
+        "\u0924\u0947\u0932\u0902\u0917\u093e\u0928\u093e \u0915\u093e \u0930\u093e\u091c\u094d\u092f"
```
Comment (nvkelso) [inline data/110/880/597/9/1108805979.geojson]: Please leave off the placetype from the preferred longname for Hindi in India (all regions). This value could be moved to an alt label? Or just ditch it.

**Comment by nvkelso** [comment]
Yes, please also do the <6 language names and labels for just the `region` level in this PR or in quick followup.

File: data/110/880/597/9/1108805979.geojson (hunk 2 above)
Comment (stepps00) [reply data/110/880/597/9/1108805979.geojson]: I'd rather just promote the Hindi name without placetype to the longname and ditch this translation. They're strung together from literal placetype translations and most references online just show the name of the region without placetype.

**Comment by stepps00** [comment]
The most recent follow up includes the following:
- resets county labels if the placetype is duplicated, removing the duplicate placetype value
- resets the Hindi region longname label to just the Hindi name (no placetype)
- Adds (up to) 14 additional longnames and placetypes for the spoken languages for all regions of India, placetypes are not included in the longname

That _should_ address the comments, but back to @nvkelso for a second glance.

**Pull request merged**